src/handlers/meta_transaction_handlers.ts

```diff
@@ -242,6 +242,8 @@ export class MetaTransactionHandlers {
                 protocolFee: metaTransactionPrice.protocolFee,
                 minimumProtocolFee: metaTransactionPrice.minimumProtocolFee,
                 allowanceTarget: metaTransactionPrice.allowanceTarget,
+                ethToInputRate: metaTransactionPrice.ethToInputRate,
+                ethToOutputRate: metaTransactionPrice.ethToOutputRate,
```

Comment:
Can we rename to `sellTokenToEth` and `buyTokenToEth` for consistency? (or eth to x), I think we'll probably forget what input/output is supposed to mean

Reply:
Yeah, that's a great idea. Will update tomorrow 👍